src/utils/Card.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import '../Components/CardListStyles.css'
+const Card = ({title, image, description}) => {
+
+  return (
+    <div className="wrapper">
+      <section className='card-container'>
+        {image ? <img src={image} alt={title} /> : 
+        <img src={'/images/SomosMas.png'} alt={title}/>}
```

Comment:
Emprolijar un poquito más este ternario para que quede más legible:

```suggestion
        {
            image ? 
                <img src={image} alt={title} /> : 
                <img src={'/images/SomosMas.png'} alt={title} />
        }
```